File: Modul 3/Soal 1/PRAK301-2410817210014-ZAVIER GHANI.py
def urutkan_angka():

    a, b, c = map(int, input("Masukkan tiga angka (dipisah dengan spasi): ").split())

    if a > b:
        a, b = b, a
    if a > c:
        a, c = c, a
    if b > c:
        b, c = c, b

    print(f"Hasil pengurutan: {a} {b} {c}")

urutkan_angka()

File: Modul 3/PRAK301-2410817210014-ZAVIERGHANI.c
#include <stdio.h>

int main() {
    int a, b, c, temp;

    printf("Masukkan tiga angka (dipisah dengan spasi): ");
    scanf("%d %d %d", &a, &b, &c);

    if (a > b) {
        temp = a;
        a = b;
        b = temp;
    }
    if (a > c) {
        temp = a;
        a = c;
        c = temp;
    }
    if (b > c) {
        temp = b;
        b = c;
        c = temp;
    }

    printf("Hasil pengurutan: %d %d %d\n", a, b, c);

    return 0;
}
